label_studio/tests/conftest.py

```diff
@@ -467,7 +467,7 @@ def project_choices():
     return {'label_config': label, 'title': 'test'}
 
 
-def setup_project(client, project_template, do_auth=True):
+def setup_project(client, project_template, do_auth=True, legacy_api_tokens_enabled=False):
```

Comment:
so we will not be using legacy tokens for any of these tests using setup_project (assume its used in a lot of places) ?

Reply:
Yeah, I tried to strike some kind of balance between migrating tests to using the new system and keeping them on the old system. Tests that seemed to be testing the token functionality more directly I kept on the old tokens, but tests that seemed agnostic I just left using the new system